docs/docs/platforms/openshift/OPENSHIFT_FAQ.md

```diff
@@ -0,0 +1,112 @@
+
+## No Pods Have Been Created
+
+This can happen when deploying Atlassian DC Helm charts to OpenShift with the default settings, and the service account has not been granted any SCC other than restricted.
+You will typically find the following error in Kubernetes events:
+
+```bash
+Error creating: pods "jira-0" is forbidden:
+unable to validate against any security context constraint:
+[provider "anyuid": Forbidden: not usable by user or serviceaccount,
+provider restricted-v2: .spec.securityContext.fsGroup: 
+Invalid value: []int64{2001}: 2001 is not an allowed group, provider "restricted":
+Forbidden: not usable by user or serviceaccount, provider "nonroot-v2":
+Forbidden: not usable by user or serviceaccount, provider "nonroot":
+Forbidden: not usable by user or serviceaccount, provider "hostmount-anyuid":
+Forbidden: not usable by user or serviceaccount, provider "machine-api-termination-handler":
+Forbidden: not usable by user or serviceaccount, provider "hostnetwork-v2":
+Forbidden: not usable by user or serviceaccount, provider "hostnetwork":
+Forbidden: not usable by user or serviceaccount, provider "hostaccess":
+Forbidden: not usable by user or serviceaccount, provider "node-exporter":
+Forbidden: not usable by user or serviceaccount, provider "efs-csi-scc":
+Forbidden: not usable by user or serviceaccount, provider "privileged":
+Forbidden: not usable by user or serviceaccount]
+```
+
+To fix this error, set `openshift.runWithRestrictedSCC` to true in Helm values.
+
+Another possible error in the events:
+
+```bash
+Error: container has runAsNonRoot and
+image will run as root (pod: "jira-0_atlassian", container: container)
+```
+
+This can happen if the ServiceAccount has been granted `nonroot` SCC privileges, but `runAsUser` has not been set in `securityContext`. To fix the error, use the following Helm values:
+
+```yaml
+jira:
+  securityContext:
+    fsGroup: 2001
+    runAsUser: 2001
+```
+
+See complete Helm values for the [Nonroot SCC](../OPENSHIFT/#__tabbed_1_2).
+
+## What's the Difference Between Restricted and Nonroot SCC
+
+From the security/compliance standpoint **Restricted SCC** enforces a more secure set of rules (or rather limitations). However, running DC containers as non root users does not expose any security threats either. The only two disadvantages is that the ServiceAccount needs to be granted nonroot SCC privileges, and there are more Helm values to be overridden. See: [Nonroot SCC](../OPENSHIFT/#__tabbed_1_2).
```

Comment:
1. Add a comma after "compliance standpoint".
2. Add a comma after "(or rather".
3. Hmmm, "non root" is here. If you're talking about the Helm charts command, make it nonroot. Otherwise, if it's general (which I'm guessing it isn't), hyphenate.